src/oop/special_class_method/hard/q10/MainTest.java

```diff
@@ -0,0 +1,199 @@
+package oop.special_class_method.hard.q10;
+
+import global.ObjectTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MainTest {
+    // Parsons with Distractors
+    private final String className = "Award";
+    private final String firstFieldName = "awardRecipient";
+    private final String secondFieldName = "awardCategory";
+    private final String setSecondMethodName = "setAwardCategory";
+    private final String getSecondMethodName = "getAwardCategory";
+    private final String getFirstMethodName = "getAwardRecipient";
+    private final String setFirstMethodName = "setAwardRecipient";
+    private ObjectTest testClass;
+
+    @BeforeEach
+    public void setUp() {
+        String classString = "oop.special_class_method.hard.q10." + className;
+        testClass = new ObjectTest(classString);
+    }
+
+    @Test
+    public void awardClassHasCorrectAttributes() {
+        assertTrue(testClass.hasField(firstFieldName),
+                "Your " + className + " class is missing the " + firstFieldName + " attributes, or it is spelled incorrectly.");
```

Comment:
these still need to be changed from "attributes" to "attribute" since we are only discussing one of the attributes